audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/kafka/KafkaContractRestTest.java

```diff
@@ -86,24 +85,24 @@
     @AfterEach
     void tearDown() {
         Arrays.stream(new File(auditServiceConfigProperties.getErrorDirectory()).listFiles())
-            .filter(file -> (file.getName().startsWith("Success") || file.getName().startsWith("Error")))
-            .peek(file -> LOGGER.info("Deleting file {}", file.getName()))
-            .forEach(File::deleteOnExit);
+                .filter(file -> (file.getName().startsWith("Success") || file.getName().startsWith("Error")))
+                .peek(file -> LOGGER.info("Deleting file {}", file.getName()))
+                .forEach(File::deleteOnExit);
```

Comment:
If we're only using the peek to write out a logger message could these be collated into the forEach method?
```suggestion
                .filter(file -> (file.getName().startsWith("Success") || file.getName().startsWith("Error")))
                .forEach((File file) -> {
                    LOGGER.info("Deleting file {}", file.getName());
                    file.deleteOnExit;
                });
```